frontend-react/src/contexts/SessionContext.tsx

```diff
@@ -36,15 +31,13 @@ const SessionProvider = ({
     oktaHook,
 }: React.PropsWithChildren<ISessionProviderProps>) => {
     const { authState } = oktaHook();
-    const store = useSessionStorage();
-    const memberships = useOktaMemberships(authState?.accessToken);
 
+    const membershipController = useOktaMemberships(authState);
     return (
         <SessionContext.Provider
             value={{
                 oktaToken: authState?.accessToken,
-                memberships,
-                store: store,
```

Comment:
UseSessionStorage has been deprecated in this change and will be removed, but I didn't do that in this PR for the sake of keeping the diff down. The next PR will be a cleanup PR that will include that removal